main.go

```diff
@@ -82,13 +86,30 @@ func (e Expo) logout() error {
 
 // Eject command creates Xcode and Android Studio projects for your app.
 func (e Expo) eject() error {
-	args := []string{"eject", "--non-interactive", "--eject-method", e.Method.String()}
+	args := []string{"eject", "--non-interactive", "--eject-method", string(e.Method)}
 
 	cmd := command.New("expo", args...)
 	cmd.SetStdout(os.Stdout)
 	cmd.SetStderr(os.Stderr)
+	if e.Workdir != "" {
+		cmd.SetDir(e.Workdir)
+	}
 
-	log.Printf("$ " + cmd.PrintableCommandArgs())
+	log.Donef("\n$ " + cmd.PrintableCommandArgs())
+	return cmd.Run()
+}
+
+func (e Expo) publish() error {
+	args := []string{"publish", "--non-interactive"}
+
+	cmd := command.New("expo", args...)
+	cmd.SetStdout(os.Stdout)
+	cmd.SetStderr(os.Stderr)
+	if e.Workdir != "" {
+		cmd.SetDir(e.Workdir)
+	}
+
+	log.Donef("\n$ " + cmd.PrintableCommandArgs())
```

Comment:
AFAIK this won't be green colored this way. Please use a `fmt.Println()` then log.Donef without the `\n`.